rpm/gnu-tar.spec

```diff
@@ -30,7 +31,6 @@ backups.
 %autosetup -p1 -n %{name}-%{version}/tar
 
 %build
-sed -i 's/doc //' Makefile.am
```

Comment:
Why is this an issue? You can run make this to avoid the patch all together:
https://github.com/sailfishos/grep/blob/master/rpm/gnu-grep.spec#L118

Reply:
It clutters up the submodule with changes, making e.g. patch-scraping fail.